django_irods/views.py

```diff
@@ -157,7 +166,7 @@ def download(request, path, rest_call=False, use_async=True, use_reverse_proxy=T
 
     # After this point, we have valid path, irods_path, output_path, and irods_output_path
     # * is_zip_request: signals download should be zipped, folders are always zipped
-    # * is_sf_agg_file: path is a single-file aggregation in Composite Resource
+    # * aggregation: aggregation object if path is to one in Composite Resource
```

Comment:
This comment above about aggregation is not clear to me - what does ```to one``` mean in this context?

Reply:
Updated to `* aggregation: aggregation object if the path matches an aggregation`